nixos/modules/system/boot/resolved.nix

```diff
@@ -138,6 +138,9 @@ in
 
     users.users.resolved.group = "systemd-resolve";
 
+    # add to nss hosts database if enabled and nscd enabled
+    system.nssDatabases.hosts = optional config.services.nscd.enable "resolve [!UNAVAIL=return]";
```

Comment:
Maybe also add here that `system.nssModules` is set in `nixos/modules/system/boot/systemd.nix`?

Reply:
Done.

Reply:
I don't think these comments are really helpful. The first line just states the obvious. Why is the second one good here @dasJ?